src/modules/bot/commands/questions/questions-interaction.collector.ts

```diff
@@ -155,12 +169,44 @@ export class QuestionsInteractionCollector {
       }, answerTimeInMilliseconds)
     }
 
+    this.stopButton.setDisabled(false)
+    const buttonRow = new ActionRowBuilder<MessageActionRowComponentBuilder>().addComponents(
+      this.watchButton,
+      this.listenButton,
+      this.stopButton
```

Comment:
We don't need to send the watch and stop button as the user has already clicked the listen button. We just need to send the stop button after the user click the listen button.